src/js/components/List/List.js

```diff
@@ -166,23 +162,25 @@ const List = React.forwardRef(
     const [orderingData, setOrderingData] = useState();
 
     const draggingRef = useRef();
-    
+
     const ariaProps = {
       role: onClickItem || onOrder ? 'listbox' : 'list',
     };
-    
-    
+
     if (active >= 0) {
       let activeId;
       // We have an item that is 'focused' within the list. This could
       // be the list item or one of the up/down ordering buttons.
       // We need to figure out an id of the thing that will be shown as active
       if (onOrder) {
         // figure out which arrow button will be the active one.
-        const buttonId = (active % 2) ? 'MoveDown' : 'MoveUp';
+        const buttonId = active % 2 ? 'MoveDown' : 'MoveUp';
```

Comment:
Interesting that the parentheses are lost in this case, just confirming that's part of the new version rules?

Reply:
Confirmed. I believe this is done by the `eslint-plugin-prettier` and the rule no-mixed-operators. Lots of folks on Google wondering why their mathematical operations loose their parens...